internal/core/convert/go.go

```diff
@@ -92,6 +90,15 @@ func parseTag(ctx *adt.OpContext, obj *ast.StructLit, field, tag string) ast.Exp
 	return expr
 }
 
+// splitTag splits a cue tag into cue and options.
+func splitTag(tag string) (cue string, options string) {
+	q := strings.LastIndex(tag, "\"")
```

Comment:
```suggestion
	q := strings.LastIndexByte(tag, '"')
```

Reply:
Why don't we also use IndexByte in line 96?
Isn't a "tag" unicode-capable and we shouldn't use `strings.*Byte` but `strings.*Rune`?

Reply:
to be clear, I only suggest the Byte forms because they're a bit simpler and faster. funcs like Index already fall back to IndexByte when the second argument is a one-byte string, but we can do that shortcut ourselves.

Reply:
I think it's okay to use IndexByte when looking for something < 0x80.
Otherwise it will fail: https://go.dev/play/p/dF8yOvB8Yjd
I could use IndexByte when looking for a comma or quotes, as utf-8 does not use bytes < 0x80 - so false positives are not expected.

Reply:
Yes, I'm pretty sure - using IndexByte or LastIndexByte is fine when your _second_ argument is plain ASCII. Your examples above would need to use the Rune forms; they break with Byte because the rune literal gets converted to byte, so information is lost.

Reply:
Yeah, that's what I meant by "< 0x80" - so if the byte is ascii only it is fine, if not it breaks.
This is thy I used Index() and not IndexByte() in the first place. But it's okay as long as we don't start using extended ascii charactes as delimiter here ;)

Reply:
I doubt we'll ever use anything other than ASCII for these separators, and if we started doing that and IndexByte broke, our tests should catch it :) if we're making syntax changes without tests, that's our fault for sure!